graph/schema.graphqls

```diff
@@ -0,0 +1,110 @@
+# Sources API GraphQL Schema
+
+# To add a field to this, do these steps:
+# 1. add the field below, with type. Docs here: https://graphql.org/learn/schema/
+# 2. run `make generate` to re-generate the gql handler
+# 3. Open up `graph/schema.resolvers.go` and implement the resolver (if required)
+# 4. PR it in!
+
+scalar Time # builtin to handle times for us
+scalar Any  # shortcut to allow `interface{}` to be returned
+
+# Filter object, where:
+# 1. name is the column and can include subresources, e.g. `source_type.vendor`
+# 2. operation is a filter operation, matching our query params. "" is eq
+# 3. value is the value to filter on
+input Filter {
+  name: String!
+  operation: String
+  value: String!
+}
+
+# validation on the direction for SortBy
+enum Direction {
+  asc, desc
+}
+
+# SortBy Object with just a field (which is required) and a direction asc/desc
+input SortBy{
+  field: String!
```

Comment:
Could we also use “name” here to be consistent with the filter?

Reply:
Absolutely, no problem.